backend/src/services/renter.ts

```diff
@@ -0,0 +1,26 @@
+import { RenterModel } from "../models/renter";
+
+//Fetch renters from DB
+export async function getRenterCandidates() {
+  const renters = await RenterModel.find({});
+  return renters;
+}
+
+export async function createRenterCandidate(
+  firstName: string,
+  lastName: string,
+  contactInfo: string,
+  program?: string,
```

Comment:
I think these may need to be updated too.

Reply:
Updated!